www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/useMousePosition.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+type MousePosition = [number, number] | null;
+
+interface UseMousePosition {
```

Comment:
```suggestion
interface MousePositionState{
```

This maybe isn't the best name, but that's the convention we used this far